sdk/trace/batch_span_processor.go

```diff
@@ -316,7 +316,10 @@ func (bsp *batchSpanProcessor) processQueue() {
 			bsp.batchMutex.Unlock()
 			if shouldExport {
 				if !bsp.timer.Stop() {
-					<-bsp.timer.C
+					select {
+					case <-bsp.timer.C:
+					default:
+					}
```

Comment:
Can you please add a comment that this is to handle currently both `asynctimerchan=0` and `asynctimerchan=1` of https://go.dev/doc/godebug.

Reply:
Sure, added